CHANGELOG.md

```diff
@@ -1,3 +1,48 @@
+## v1.1.0
+
+### Additions
+
+- For Ansible-based Operator, add `.gitignore` file. ([#3806](https://github.com/operator-framework/operator-sdk/pull/3806))
+- Added the `--select-optional` flag to `bundle validate` to pass a label selector that runs any matching optional validator on top of required validators, and `--list-optional` to display optional validators, their selectors, and a short explanation of what each does. ([#3719](https://github.com/operator-framework/operator-sdk/pull/3719))
+- Added the OperatorHub.io optional validator to `bundle validate`, which can be run by setting `--select-optional=name=operatorhub` or `--select-optional=suite=operatorframework`. ([#3719](https://github.com/operator-framework/operator-sdk/pull/3719))
+- Added the `run bundle` command. ([#3988](https://github.com/operator-framework/operator-sdk/pull/3988))
+- Added new base image assets containing the Operator-SDK CLI at quay.io/repository/operator-framework/operator-sdk. ([#3902](https://github.com/operator-framework/operator-sdk/pull/3902))
+- Removed stale unix sockets. This allows for smoother  initialization during the start-up of an Ansible operator,  as each operator will clean up its sockets during termination. ([#3721](https://github.com/operator-framework/operator-sdk/pull/3721))
+- Added feature to the command `operator-sdk olm install` that allows installing OLM version 0.15.1 without  fetching the manifest from Github. ([#3906](https://github.com/operator-framework/operator-sdk/pull/3906))
+- Re-enabled s390x docker image builds. ([#3855](https://github.com/operator-framework/operator-sdk/pull/3855))
+
+### Changes
+
+- In `ansible-operator` and `helm-operator` run commands, print git commit when logging version information. ([#3849](https://github.com/operator-framework/operator-sdk/pull/3849))
+- For Ansible/Helm, fix multigroup message. ([#3822](https://github.com/operator-framework/operator-sdk/pull/3822))
+- For Helm-based Operators, cleanup the file `.gitignore` in order to not have invalid instructions for the type. ([#3810](https://github.com/operator-framework/operator-sdk/pull/3810))
+- Updated scorecard-test-kuttl image to kuttl v0.6.1. ([#3659](https://github.com/operator-framework/operator-sdk/pull/3659))
+- Updated scorecard-test-kuttl image to kuttl v0.5.2. ([#3659](https://github.com/operator-framework/operator-sdk/pull/3659))
+- Updated scorecard-test-kuttl image to use latest kuttl. ([#3711](https://github.com/operator-framework/operator-sdk/pull/3711))
+- Updated the Makefile to build multiarch versions of the scorecard-test-kuttl and custom-scorecard-tests images to match the scorecard-test image. ([#3821](https://github.com/operator-framework/operator-sdk/pull/3821))
+
+### Bug Fixes
```

Comment:
Only Missing  https://github.com/operator-framework/operator-sdk/pull/3983




12:38

Reply:
3983 is now included.